Dockerfile

```diff
@@ -1,5 +1,4 @@
 FROM ubuntu/squid:latest
-VOLUME ["/docker/custom-certs"]
```

Comment:
Is this change tested? I had some permission issues in another project then running the entrypoint script, because the directory wasn't previously created.